### ISSUE TITLE
Rendering the table in "view"

### ISSUE THREAD

**Issue by enboig** [ThibaultJanBeyer/joplin-sheets#15]
Could the edited cells be rendered as a table in "view"?

**Comment by everdred** [comment]
I was just wondering this, myself. It would be especially nice to have for viewing on Joplin mobile.

**Comment by tomcollis** [comment]
I don't think this is possible, as joplin-sheets is purely storing the data necessary for luckysheets in a code block, and more complex sheets wouldn't work well as a static "view".
